### PR TITLE
[Components] Add sistent naming convention for our components

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #

For being consistent we can use the sistent naming convention instead of mui for exporting these  theme based components.

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
> So we're only just naming the custom components, not the exported components to be used in Meshery?

They might because i think these styled component will only play major role when wrapped with sistent provider right!.
The naming is for only to keep our sistent codebase looks like its our own components and colors.

File: packages/components/src/theme/components/checkbox.modifier.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Components, Theme } from '@mui/material';
 import { CHARCOAL, connected, darkSlateGray } from '../colors';
 
-export const MuiCheckbox: Components<Theme>['MuiCheckbox'] = {
+export const SistentCheckbox: Components<Theme>['MuiCheckbox'] = {
```
Comment (aabidsofi19) [inline packages/components/src/theme/components/checkbox.modifier.ts:4]: @sudhanshutech  using sistent as prefix is redundant ( as they will be scoped under sistent ) , I would suggest use `Base-<Component>` for unstyled / customizable component and `<Functionality-<Component>` for customized and ready to use ones  , @nebula-aac  whats your opinion ?

**Review by aabidsofi19 — CHANGES_REQUESTED** [review]

**Comment by aabidsofi19** [comment]
@sudhanshutech  @nebula-aac  I would suggest using our custom convention for both private and public components

File: packages/components/src/theme/components/checkbox.modifier.ts (hunk 1 above)
Comment (aabidsofi19) [reply packages/components/src/theme/components/checkbox.modifier.ts:4]: @nebula-aac  @sudhanshutech    we dont need to have a complex naming convention , lets agree of something simple and stay consistent

**Pull request closed without merging**